src/main/drivers/system.h

```diff
@@ -45,9 +45,9 @@ void failureMode(failureMode_e mode);
 // bootloader/IAP
 void systemReset(void);
 void systemResetToBootloader(void);
+void checkForBootLoaderRequest(void);
```

Comment:
Any reason for the reordering?

Reply:
To keep it logically structured and in line with implementation file

Reply:
I'd rather have less unneeded churn. The ordering of of declarations in the header file is independent of the ordering in the implementation, and refactoring the implementation should not affect the public API.